identity/webapp/src/frontend/MyAccount/MyAccount.tsx

```diff
@@ -53,14 +66,38 @@ const AccountStatus: React.FC<React.ComponentProps<typeof StatusAlert>> = ({ typ
   );
 };
 
+async function fetchRequestedItems(userId): Promise<RequestsList | undefined> {
+  try {
+    const response = await fetch(`/api/users/${userId}/item-requests`);
+    const json = await response.json();
+    return json;
+  } catch (e) {
+    console.log(e);
+  }
+}
+
 const Profile: React.FC = () => {
   const history = useHistory();
   const { user, isLoading, update } = useUserInfo();
   const [isEmailUpdated, setIsEmailUpdated] = useState(false);
   const [isPasswordUpdated, setIsPasswordUpdated] = useState(false);
+  const [requests, setRequests] = useState<RequestsList | undefined>();
```

Comment:
```suggestion
  const [requests, setRequests] = useState<RequestsList>();
```

undefined is inferred as there is no value.

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L923
vs
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L915